frontend/packages/dev-console/integration-tests/features/pipelines/event_tab_pipeline_run_page_ODC_4314

```diff
@@ -0,0 +1,72 @@
+Feature: Events tab in Pipeline run and Task run details pages
+    As a user, I want to see Events tab in Pipeline run and Task run details pages
+
+Background:
+    Given user has installed OpenShift Pipelines Operator
+
+
+@regression
+Scenario: Events tab in pipeline run details page of administrator perspective
+   Given user is at administrator perspective
+   And pipeline "<pipeline_name>" is available with pipeline run
+   When user clicks on pipeline tab in navigation menu
+   And user clicks on Pipelines
+   And user goes to Pipeline Runs tab
+   And user opens pipeline run
+   And user clicks on Events tab
+   Then user can see events streaming for pipeline runs and all the associated task runs and pods
+
+Examples:
+| pipeline_name   |
+| example-pipeline_one |
+
+
+@regression
+Scenario: Events tab in task run details page of administrator perspective
```

Comment:
@sanketpathak - If you want to use Examples, Scenario Outline should be mentioned
Please refer https://cucumber.io/docs/gherkin/reference/#:~:text=A%20Scenario%20Outline%20must%20contain,counting%20the%20first%20header%20row).